### PR TITLE
Adjust testerina guide to ballerina 1.0 syntax

### PR DESCRIPTION
**Pull request by rdhananjaya**
## Purpose
$subject 

- Any checklist items to be verified before production deployment?
- Special styling concerns? 
- Related media files?

### CODE REVIEW

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 1]
```diff
@@ -105,14 +91,14 @@ function beforeFunc() {
...
-@test:Config
+@test:Config{}
 function testFunction1() {
     io:println("I'm in test function 1!");
     test:assertTrue(true, msg = "Failed");
 }
 
 // Test function.
-@test:Config
+@test:Config{}
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]:
```suggestion
@test:Config {}
```

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 2]
```diff
@@ -142,21 +128,21 @@ function beforeFunc() {
 }
 
 // Test function
-@test:Config
+@test:Config{}
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]:
```suggestion
@test:Config {}
```

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 3]
```diff
@@ -142,21 +128,21 @@ function beforeFunc() {
...
 function testFunction1() {
     io:println("I'm in test function 1!");
     test:assertTrue(true, msg = "Failed!");
 }
 
 // Test function
-@test:Config
+@test:Config{}
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]:
```suggestion
@test:Config {}
```

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 4]
```diff
@@ -142,21 +128,21 @@ function beforeFunc() {
...
 function testFunction2() {
     io:println("I'm in test function 2!");
     test:assertTrue(true, msg = "Failed!");
 }
 
 // Test function
-@test:Config
+@test:Config{}
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]:
```suggestion
@test:Config {}
```

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 5]
```diff
@@ -229,7 +209,7 @@ function testFunction1() {
 
 // This is a random test function, this will randomly execute without depending on other functions.
 // But note that other function do depend on this.
-@test:Config
+@test:Config{}
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]:
```suggestion
@test:Config {}
```

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 6]
```diff
@@ -254,7 +234,7 @@ import ballerina/io;
 import ballerina/test;
 
 // Test function.
-@test:Config
+@test:Config{}
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]:
```suggestion
@test:Config {}
```

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 7]
```diff
@@ -276,7 +256,7 @@ Asserts that the expression is true with an optional message.
 ```ballerina
 import ballerina/test;
 
-@test:Config
+@test:Config{}
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]:
```suggestion
@test:Config {}
```

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 8]
```diff
@@ -290,7 +270,7 @@ Asserts that the expression is false with an optional message.
...
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]:
```suggestion
@test:Config {}
```

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 9]
```diff
@@ -304,7 +284,7 @@ Asserts that the actual is equal to the expected, with an optional message.
...
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]:
```suggestion
@test:Config {}
```

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 10]
```diff
@@ -326,7 +306,7 @@ Asserts that the actual is not equal to the expected, with an optional message.
...
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]:
```suggestion
@test:Config {}
```

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 11]
```diff
@@ -348,7 +328,7 @@ Fails the test. Useful when we want to fail a test while in execution based on a
 ``` ballerina
 import ballerina/test;
 
-@test:Config
+@test:Config{}
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]:
```suggestion
@test:Config {}
```

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 12]
```diff
@@ -180,13 +166,7 @@ Default: none
...
 
 `groups: [“<test group name”, …]`:
-List of test group names (one or more) that this test belongs to. You can group a given test to a list of named test groups using this configuration. In order to execute tests belonging to a selected test group, you can name the test groups to be executed when you run tests.  
-
-```
-ballerina test --groups <comma separated list of test group names> <module_name>
-```
-
-You can skip a list of given tests with `--disable-groups <comma separated list of test group names>` Also you can use the  `--list-groups` flag to list the groups in your tests.
+List of test group names (one or more) that this test belongs to. You can group a given test to a list of named test groups using this configuration.
```
Comment (pubudu91) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]: The heading of this section needs to be renamed.. "Parameters" => "Annotation Value Fields"

Comment (MaryamZi) [reply website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]: Will send a PR addressing this.

**Pull request merged**